packages/tools/webpack-fluid-loader/src/multiDocumentServiceFactory.ts

```diff
@@ -26,6 +26,10 @@ export function getDocumentServiceFactory(documentId: string, options: RouteOpti
         name: getRandomName(),
     });
 
+    if (options.mode === "tinylicious") {
```

Comment:
Don't add these to options. You can probably create local variables for tenantId and tenantSecret and give them values based on the mode. Then pass those to the InsecureTokenProvider.

Reply:
Ditto